src/bool.cr

```diff
@@ -55,4 +55,9 @@ struct Bool
   def to_s(io)
     io << to_s
   end
+
+  # Returns `self`
```

Comment:
I wouldn't hide them from the API docs, but no comment is fine with me. "Returns `self`" is misleading if you come from a pass by reference only language IMO.